### PR TITLE
Feat: Activate/deactivate surveys

### PR DESCRIPTION
**Pull request by umarphaarook**
## Description
Feature that allow users to activate and deactivate surveys

### CODE REVIEW

File: app/gather/assets/apps/survey/SurveysList.jsx [hunk 1]
```diff
@@ -18,40 +18,29 @@
...
-import SurveyCard from './SurveyCard'
+import ActiveSurveys from './ActiveSurveys'
+import InactiveSurveys from './InactiveSurveys'
 
-export default class SurveysList extends Component {
-  render () {
-    const { list } = this.props
+const SurveysList = ({ list }) => {
+  if (list.length === 0) return <div data-qa='surveys-list-empty' />
+  console.log('LIST => ', list)
```
Comment (obdulia-losantos) [inline app/gather/assets/apps/survey/SurveysList.jsx]: :grimacing:

File: app/gather/assets/apps/survey/SurveysList.jsx [hunk 2]
```diff
@@ -18,40 +18,29 @@
...
-        <h4 className='title'>
-          <FormattedMessage
-            id='survey.list.title'
-            defaultMessage='Surveys'
-          />
-        </h4>
+  list.forEach(survey => {
+    if (!survey.active) activeSurveys.push(survey)
+    else inactiveSurveys.push(survey)
+  })
```
Comment (obdulia-losantos) [inline app/gather/assets/apps/survey/SurveysList.jsx]: Or...

```javascript
const activeSurveys = list.filter(({ active }) => active)
const inactiveSurveys = list.filter(({ active }) => !active)
```
or...
```javascript
const SurveysList = ({ list }) => list.length
  ? (
    <div data-qa='surveys-list' className='surveys-list'>
      <ActiveSurveys list={list.filter(({ active }) => active)} />
      <InactiveSurveys list={list.filter(({ active }) => !active)} />
    </div>
  )
  : <div data-qa='surveys-list-empty' />
```

**Comment by shawnsarwar** [comment]
@umaroolfaruk What's the status here?

**Comment by obdulia-losantos** [comment]
> @umaroolfaruk What's the status here?

I'll make the last requested changes on this PR:

**Comment by obdulia-losantos** [comment]
With pagination bar in both sections

image

**Comment by obdulia-losantos** [comment]
Without data
image

**Comment by obdulia-losantos** [comment]
One active survey
image

One inactive survey
image

**Review by shawnsarwar — APPROVED** [review]

**Pull request merged**